lib/ansible_tower_client/base_models/workflow_job_node.rb

```diff
@@ -5,7 +5,7 @@ def workflow_job
     end
 
     def job
-      job_id.nil? ? nil : api.jobs.find(job_id)
+      api.jobs.find(job_id) if respond_to?(:job_id) && job_id
```

Comment:
contributed by @bdunne [here](https://github.com/ansible/ansible_tower_client_ruby/commit/21c22ef3e62c663210a12633cd67d75fc3ed859a#diff-bc25d095695ec0bc4c9c49a9e4ef0134)

Reply:
I think we might just want @bdunne's raw commit - the mock object isn't needed and rubocop will clear up as well.